.gitignore

```diff
@@ -7,3 +7,8 @@ deps/deps.jl
 docs/build/
 docs/site/
 src/**/Manifest.toml
+*.vtk
+*.vtu
+*.swp
```

Comment:
I'm happy to keep this here, but it really should go in your personal `.gitignore` file. I think the default path is
`~/.config/git/ignore`

cc: @asraero

Reply:
Noted - .gitignore locally has been updated